examples/base/water-river-example.yaml

```diff
@@ -0,0 +1,24 @@
+---
+type: Feature
+geometry:
+  type: LineString
+  coordinates: [[-91.8307764, 43.7899936], [-91.8309309, 43.7901326], [-91.8315202, 43.7906232], [-91.8320790, 43.7911798], [-91.8325770, 43.7917638], [-91.8328390, 43.7921615]  ]
+properties:
+  theme: base
+  type: water
+  subType: river
+  class: river
+  names:
+    common:
+      - value: Root River
+        language: local
+  sourceTags:
+    waterway: river
+  sources:
+    - recordId: w54320702@1
+      property: ""
+      dataset: OpenStreetMap
+  isSalt: false
+  isIntermittent: false
```

Comment:
For post M1, we could consider following the transportation segment model and having a flags array here. Would keep the column count down a bit.